Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -78,15 +78,15 @@ public class ComputeCredential : ServiceCredential, IOidcTokenProvider, IGoogleC
             /// <summary>Constructs a new initializer using the default compute token URL
             /// and the default OIDC token URL.</summary>
             public Initializer()
-                : this(GoogleAuthConsts.ComputeTokenUrl) {}
+                : this(GoogleAuthConsts.EffectiveComputeTokenUrl) {}
 
             /// <summary>Constructs a new initializer using the given token URL
             /// and the default OIDC token URL.</summary>
             public Initializer(string tokenUrl)
-                : this(tokenUrl, GoogleAuthConsts.ComputeOidcTokenUrl) {}
+                : this(tokenUrl, GoogleAuthConsts.EffectiveComputeOidcTokenUrl) {}
 
             /// <summary>Constructs a new initializer using the given token URL
-            /// and OIDC token URL.</summary>
+            /// and OIDC token URL (optionally overriding the host using the GCE_METADATA_HOST environment variable).</summary>
```

Comment:
I think the "(optionally ...)" should be on the above constructur, the one that uses the EffectiveComputeOidcTokenUrl, as this one does not override anything, it just uses whatever is passed as parameter.

Reply:
Yes, absolutely. And I've added a bit on the parameterless constructor too.